Makefile

```diff
@@ -29,3 +31,10 @@ bdd-test: clean
 .PHONY: clean
 clean:
 	rm -f coverage.txt
+	rm -Rf ./build
+
+.PHONY: validate-api-spec
```

Comment:
I think we should be more explicit that this is an openapi/swagger target

validate-openapi-spec or validate-openapi

Reply:
Also I see below that the openapi is generated.

Are you sure we shouldn't call the target generate-foo?

Reply:
> I think we should be more explicit that this is an rest/openapi/swagger target
> 
> e.g., validate-openapi-spec, validate-openapi, ...

done

Reply:
> Also I see below that the openapi is generated.
> 
> Are you sure we shouldn't call the target generate-foo?

we generate the spec only to feed it into validate command, we are not yet including spec in codebase. Overall objective of this target it to validate the rest api code against open API specifications.

Reply:
Would someone want to consume the generated specs separate from validation?

Reply:
Good question - and it's what I understood from our last conversation.